Wording should unambiguously match implementation (#21232)
mdn/content: files/en-us/web/javascript/reference/global_objects/array/findlast/index.md

{{JSRef}}

> The **`findLast()`** method returns the value of the first element from the end of an array that satisfies the provided testing function.

Reviewer: It took me a few reads to understand your point, but you're right. Should we "spell it out" even more?

```suggestion
The **`findLast()`** method iterates the array in reverse order and returns the value of the first element that satisfies the provided testing function.
```

Can you please also update `findLastIndex` and the matching `TypedArray` methods?

Author: Excellent! I've committed changes using your wording for the `findLast` and `findLastIndex` methods of both `Array` and `TypedArray`. I wasn't totally satisfied with my wording, so thank you for your suggestion. I wonder if someone might still get confused about what "first" means, though?

> The **`findLast()`** method iterates the array in reverse order and returns the value of the first element that satisfies the provided testing function.

Reviewer: > I wonder if someone might still get confused about what "first" means, though?

@jdanhutch I don't think there is any ambiguity here now - it's the "first element that matches the criteria when searching in reverse order", not "the first element in the array". @teoli2003 do you have an opinion/see a better way?
